doc/tests/block.rst

```diff
@@ -0,0 +1,13 @@
+``block``
+=========
+
+.. versionadded:: 1.23
+    The ``block`` test was added in Twig 1.23.
+
+``block`` checks if a block is defined in the current context of the template.
+
+.. code-block:: jinja
+
+    {% if 'title' is block %}
```

Comment:
I prefer a test instead of a function. In #1821 I suggested ``{% if 'title' is block name %}``, for example.